test-suite/src/test/scala/org/scalajs/testsuite/javalib/StringTest.scala

```diff
@@ -248,5 +248,40 @@ object StringTest extends JasmineTest {
               -94, 22, -41))
     }
 
+    it("should respond to `regionMatches`") {
+      // Ported from https://github.com/gwtproject/gwt/blob/master/user/test/com/google/gwt/emultest/java/lang/StringTest.java
+      val test = String.valueOf(Array('a', 'b', 'c', 'd', 'e', 'f'))
```

Comment:
yep - I saw this too. It's so ..."weird" that I couldn't help think that it was done for a purpose, perhaps to use the lowest String constructor (as it's in the String class tests). Or could just be very old code. 

I'll change it though, unless you say otherwise